service/implementation/auto_request_api/sport_data_managers/teams_data_manager.py

```diff
@@ -0,0 +1,54 @@
+from typing import Optional
+
+from database.models import Sport
+from database.session import SessionLocal
+from dto.api_input import TeamsLeagueDTO
+from service.api_logic.teams_logic import get_teams
+from service.implementation.auto_request_api.sport_data_managers.abstract_sport_data_manager import \
+    AbstractSportDataManager
+from service.implementation.auto_request_api.sport_data_managers.sport_consts import get_team_statistics_url, \
+    get_team_statistics_index, get_host, get_team_index
+
+
+class TeamsDataManager(AbstractSportDataManager):
+    _teams__sport_id: Optional[int]
+    _leagues__api_id: Optional[int]
+    _countries__api_id: Optional[int]
+    _page: Optional[int]
+    _per_page: Optional[int]
+
+    def __init__(self, data_manager):
+        super().__init__(data_manager)
+        query = (
+            SessionLocal().query(
+                Sport.sport_id,
+                Sport.sport_name
+            )
+            .filter(Sport.sport_id == data_manager.get("teams__sport_id"))
+        )
+
+        ix = query.first()
+        if ix is not None:
+            self._teams__sport_id = ix.sport_name
+        #self._teams__sport_id = data_manager.get("teams__sport_id")
+        self._leagues__api_id = data_manager.get("leagues__api_id")
+        self._host = get_host(self._teams__sport_id)
+
+        #self._sport_name = get_sport_name(self._teams__sport_id)
+
+        #self._countries__api_id = data_manager.get("countries__api_id")
+        self._page = data_manager.get("page")
+        self._per_page = data_manager.get("per_page")
```

Comment:
MINOR:

I would make the names slightly more expressive:

page_number
elements_per_page

something like that

tou always want your code to be as self-explanatory as you possibly can (no one wants to launch their shit two months later in order to debug some freaking line noone remebers ever being written in the first place just to check what it does)